client/layout/logged-out.jsx

```diff
@@ -50,6 +56,18 @@ const LayoutLoggedOut = ( {
 	isPartnerSignupStart,
 	locale,
 } ) => {
+	const dispatch = useDispatch();
+	const isStateRandomlyCleared = useSelector( getIsStateRandomlyCleared );
+
+	useEffect( () => {
+		if ( isStateRandomlyCleared ) {
+			setTimeout( () => {
+				dispatch( sympathyWarningNotice() );
+				dispatch( developerNotificationCompleted() );
+			}, 1200 );
```

Comment:
Added a delay so the notice appears after the UI is mostly loaded.

Reply:
Are we certain that this delay will be enough for smaller machines? Should we use different means to ensure that UI has been loaded?

Reply:
This is not an end user facing feature and I didn't think it's vital to ensure this. Maybe I can increase the duration the notice lingers on before auto disappearing?